python/apertium_core.i

```diff
@@ -89,3 +131,13 @@ public:
   void postchunk_text(char arg, char *transferfile, char *datafile, char *input_path, char *output_path);
   void transfer_text(char arg, char *transferfile, char *datafile, char *input_path, char *output_path);
 };
+
+class apertium_tag: public Apertium::apertium_tagger
```

Comment:
Is there a reason we're using `apertium_tag` instead of `apertium_tagger` here?

Reply:
`apertium_tagger` expects `int&` and `char**&`. I tried wrapping `char**&` but that didn't work out and I have no idea how to wrap `int&`, so I used a class to pass the required arguments

Reply:
Hmm, okay. I was referring more to the name itself. Can it also be `apertium_tagger` or will that cause conflicts?

Reply:
> I was referring more to the name itself. Can it also be `apertium_tagger` or will that cause conflicts?

Renaming `tag` to `apertium_tagger` is causing conflicts `Error: Previous declaration of 'apertium_tagger'`

Reply:
Ah. Can we just call it `tagger` then? Presumably it'll be used as `apertium.tagger` anyway? We might want to make a similar change for postchunk, interchunk, etc.

Reply:
> Ah. Can we just call it `tagger` then?

OK


> Presumably it'll be used as `apertium.tagger` anyway?

This would be the command used for calling tagger, after the rename, `apertium_core.tagger(len(command), command)` https://github.com/apertium/apertium-python/blob/swig_wrapper_tagger/apertium/utils.py#L82. If we want to create a member function which invokes the tagging process, the `apertium_tagger` would have to be split into two functions.


> We might want to make a similar change for postchunk, interchunk, etc.

Is this regarding the support of multiple arguments? I am thinking of using getopts in the wrapper,to parse all the arguments.

Reply:
> Is this regarding the support of multiple arguments? I am thinking of using getopts in the wrapper,to parse all the arguments.

No, I was just commenting regarding the naming scheme.

> This would be the command used for calling tagger, after the rename, `apertium_core.tagger(len(command), command)`

Yeah. That's better than `apertium_core.apertium_tagger`. The latter is pretty repetitive. We're treating the wrappers as a private API for now so it doesn't matter too much.